### ISSUE TITLE
Asbuilt attribute manager (TMDL Widget) - 6.58.6 fields/attributes

### ISSUE THREAD

**Issue by rmeitl** [dhenry-KCI/TMDL-_Tracking-Widget#41]
--Submittal Date should pull data from SUBMITTAL_DATE

--Review Date should pull data from REVIEW_DATE

--Reviewer should pull data from REVIEWER field

--And Consultant should pull data from REVIEWER_CONSULTANT_ID - while using CONSULTANT as a look up table

**Comment by ecomwu** [comment]
I do have questions about this issue.
- For the first 2 items, currently, Submittal Date, Review Date, and Reviewer are all pull data from correct fields. If there is any data display issue, could you provide the specific selection to recreate the issue?
- For the last 2 items, there is only REVIEWER_CONSULTANT_ID listed in data return and no REVIEWER field.

**Comment by dhenry-KCI** [comment]
![as_built](https://cloud.githubusercontent.com/assets/4129185/3389141/64732e18-fc91-11e3-9f8d-f9cf00486202.JPG)
Maggie,
1. The submittal date and the review date are not displaying in the widget. I have included a screen shot showing the issue. I used record Tree planting record 160087UT to recreate the issue.
2. The reviewer field is included in the database and is a field in the widget. I will Ehsan know that he may need to write a service for this field.
